habitat/plan.sh

```diff
@@ -0,0 +1,59 @@
+pkg_name=knife-ec-backup
+pkg_origin=chef
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="A knife plugin for backup/restore of Chef Server data."
+pkg_upstream_url="https://github.com/chef/knife-ec-backup"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+pkg_svc_user=root
+pkg_svc_group=${pkg_svc_user}
+pkg_build_deps=(
+  core/gcc-libs
+  core/libffi
+  core/make
+  core/openssl
+)
+pkg_deps=(
+  core/bundler
+  core/coreutils
+  core/gcc
+  core/ruby
+)
+
+pkg_build_deps=(
+  core/git
+  core/make
+  core/postgresql-client
+)
+
+do_unpack() {
+  mkdir -p "${HAB_CACHE_SRC_PATH}/${pkg_dirname}"
+  cp -r ../* ${HAB_CACHE_SRC_PATH}/${pkg_dirname}/
```

Comment:
I would probably wrap this command in a sub-shell:
```bash
( cd $FOO | exit_with "unable to enter foo" 1
  cp -r * ${HAB_CACHE_SRC_PATH}/${pkg_dirname}/
)
```

Another option is using absolute paths to about the sub-shelling:
```bash
  cp -r "$FOO/"* "${HAB_CACHE_SRC_PATH}/${pkg_dirname}/"
```